Makefile

```diff
@@ -104,21 +101,24 @@ install-torch-test:
 	pip install -r examples/torch/requirements.txt
 
 install-torch-dev: install-torch-test install-pre-commit install-pylint
+	pip install -r examples/post_training_quantization/torch/ssd300_vgg16/requirements.txt
+	pip install -r docs/api/requirements.txt
```

Comment:
Not needed

Reply:
I added it to cover all requirements available for torch backend, to not miss anything if any new requirement come to this files

Reply:
I only meant the `docs/api/requirementst.txt` line.

Reply:
Ok, I did it